README.md

```diff
@@ -1,64 +1,192 @@
-# Pattern Lab Node - Gulp Edition
+# PatternFly 4 CSS
 
-The Gulp wrapper around [Pattern Lab Node Core](https://github.com/pattern-lab/patternlab-node) providing tasks to interact with the core library and move supporting frontend assets.
+**PatternFly 4** is the best open source modular design system you'll ever use to build enterprise web applications.
+
+Think of PatternFly as a "skinned" version of [Bootstrap v4](http://v4-alpha.getbootstrap.com/) with additional components and customizations.
+
+This repository is javascript free, you'll find only find CSS here.
+
+## Are we there yet?
+
+No, we are not there yet. PatternFly 4 is in its very early stages.
+
+If you as passionate about CSS as we are, you would love to participate in the community and became a contributor.
+
+### Communication channels
+
+You can contact the patternFly community on the [PatternFly mailing list](https://www.redhat.com/mailman/listinfo/patternfly) or chat with us on the [PatternFly slack channel](https://patternfly.slack.com/).
+
+
+## Contributing to PatternFly 4
+
+PatternFly 4 is implemented using [Pattern Lab Node - Gulp Edition](https://github.com/pattern-lab/patternlab-node).
+
+### Coding style guidelines
+
+Please read the coding style guidelines for PatternFly 4.
+
+The guidelines are a work in progress and a live document, if you have any suggestion please [contact the PatternFly community](#communication-channels) or send a PR.
```

Comment:
Another semicolon - or maybe even better, a period - instead of comma here. Also, "suggestions" should be plural.